### DISCUSSION TITLE
5.2.3 BASIC NOTIFICATION INTERFACE - NOTIFY Blocking dialog

### DISCUSSION THREAD

**Discussion by pawelbialek** [onvif/testspecs#85] [Q&A] [answered]
Good day,

We are trying to automate the ONVIF profile conformance test, however without any property events provided we are being displayed the following dialog every time a wait for an event starts in ODTT:
![image](https://github.com/user-attachments/assets/7bb1d550-f7e7-4096-8acd-8e2641bec3ed)
Is having some mock property event in `GetEventProperties` so that this dialog is shown the only way to automate the process ?
Isn't there a way to disable the dialog completely on ODTT and assume the device generates the events when they are needed ?

thanks in advance,

- Pawel

**Comment by madhurao68** [comment, marked as answer]
Hello,

The DTT has a feature to automate the user interactions during conformance. You will find the details in the Help in the section Functional Overview/ Management Tab/ Operator Actions File. On the UI you will find an _Operator Actions _ sub-tab under the Management tab which takes a script file for executing operator actions. The DTT executes the script corresponding to the test and executes the application. There is a sample script in the Help as well. Even if one does not want to run a real a real script for event generation, you can use this functionality with a fake script - in this case the UI window will not be shown during conformance.

Additionally, you can look at the Command Line Mode section of the Help (located under Functional Overview) for more on the automation process.

![image](https://github.com/user-attachments/assets/0b54080f-a175-44b8-9b56-55802be17486)

**Reply by pawelbialek** [reply]
Many thanks ❤️
